plugin.video.drnu/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.drnu" version="5.0.10" name="DR TV" provider-name="twinther">
+<addon id="plugin.video.drnu" version="5.0.12" name="DR TV" provider-name="msj33">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
> For gotham it is advised to set xbmc.python version to 2.14.0
```suggestion
         <import addon="xbmc.python" version="2.14.0"/>
```

Reply:
Bumped the Python version as recommended